docs/diga_api.md

```diff
@@ -0,0 +1,216 @@
+# Overview
+
+Manufacturers of digitial health applications (DiGA) can apply for a fast track at [bfarm](https://www.bfarm.de/EN/MedicalDevices/DiGA/_node.html) to place their solutions in a directory of apps which can be prescribed by doctors to patients. By showing evidence that the apps improve patients health and by obtaining medical device certifications, it can be assured that the apps are of high quality. On the other side, DiGA manufacturers get reimbursed by health insurances when their apps are prescribed to patients. The prescription process is build around a 16-character prescription code which can be used to activate the apps but which is also required to send the invoices. This repository holds code to:
+
+- verify that user-entered prescription codes are valid
+- create and send invoices to insurance companies based on prescription codes.
+
+This documentation complements the code by describing the DiGA api and the full flow on a higher level. With this documentation we hope that other DiGA manufacturers can understand the required steps to verify and reimburse prescription codes without having to read through the (mainly German) official documentation. Also we hope that it will facilitate writing similar solutions in other programming languages if necessary. Finally, it can be used as a place to collect questions and answers around the DiGA api.
+
+## Summary
+
+- user enters prescription code in the DiGA (app)
+- insurance information is extracted from the code
+- a request is made to the api of the insurance to verify the code
+  - response contains `Tag der Leistungserbringung` which is the day a user started to use the app
+  - this is required for the billing process
+- another request is made for reimbursing the code
+
+## The prescription code
+
+Patients receive 16-character prescription codes by their insurances. The codes are used to activate the DiGA apps and for creating invoices for billing. The structure of the code is defined [here](https://www.gkv-datenaustausch.de/media/dokumente/leistungserbringer_1/digitale_gesundheitsanwendungen/technische_anlagen_aktuell_7/DiGA_Anlage_1_Technische_Anlage_zur_RL_V1.1_20210225.pdf) on page 9 (German).
```

Comment:
Perhaps better to link to the page and the`Anlage 1 ` header. The `pdf` contains the day it was published so the link will become outdated quickly.